backend/climateconnect_api/utility/email_setup.py

```diff
@@ -1,19 +1,27 @@
 import logging
```

Comment:
I would recommend that we separate this file into multiple files. 

Create `email_setup` folder under `utility` folder. Then separate methods according to what they do.

Reply:
My preference is that if the file has more than 800 lines of code try to separate them into smaller files so it is easier for other engineers to read and manage code.

Reply:
Thanks, will remember that!